tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -300,6 +300,7 @@ def test_torch_large(self):
         self.assertEqual(output, {"text": "A MAN SAID TO THE UNIVERSE SIR I EXIST"})
 
     @require_torch
+    @slow
```

Comment:
Looks like this test is not related to the commit that was merged yesterday, which didn't touch this test: https://github.com/huggingface/transformers/commit/8d518013efbd10c178dd0dba0f9ba93229e2e78a

This probably failed due to using the `whisper-tiny` model in the test, rather than a `hf-internal-testing` variant

Went through all the ASR pipeline tests and they're all correctly marked as slow / not slow now 👍